File: src/main/java/com/indico/entity/Workflow.java
package com.indico.entity;

public class Workflow {

    public final int id;
    public final String name;
    public final boolean reviewEnabled;

    private Workflow(Builder builder) {
        this.id = builder.id;
        this.name = builder.name;
        this.reviewEnabled = builder.reviewEnabled;
    }

    public static class Builder {

        protected int id;
        protected String name;
        protected boolean reviewEnabled;

        public Builder id(int id) {
            this.id = id;
            return this;
        }

        public Builder name(String name) {
            this.name = name;
            return this;
        }

        public Builder reviewEnabled(boolean reviewEnabled) {
            this.reviewEnabled = reviewEnabled;
            return this;
        }

        public Workflow build() {
            return new Workflow(this);
        }
    }
}

File: bin/main/com/indico/graphql/listworkflowsgraphql/Workflow.kt
package com.indico.graphql.listworkflowsgraphql

import com.expediagroup.graphql.client.Generated
import kotlin.Boolean
import kotlin.Deprecated
import kotlin.Int
import kotlin.String

@Generated
public data class Workflow(
  /**
   * Unique ID of the workflow
   */
  public val id: Int?,
  public val name: String?,
  /**
   * DEPRECATED: Status of the Review queue
   */
  @Deprecated(message = "Now uses settings' `review_queue_enabled`")
  public val reviewEnabled: Boolean?
)
